pkg/cloud-init/cloud-init_test.go

```diff
@@ -33,8 +33,8 @@ import (
 	. "github.com/onsi/gomega"
 
 	k8sv1 "k8s.io/api/core/v1"
-
```

Comment:
nit: This is not a related change. We also prefer to keep different domains separated.